saleor/graphql/webhook/subscription_types.py

```diff
@@ -1157,12 +1157,35 @@ def resolve_order(root, info: ResolveInfo):
 
 
 class FulfillmentCreated(SubscriptionObjectType, FulfillmentBase):
+    notify_customer = graphene.Boolean(
+        description=(
+            "If true, send an email notification to the customer." + ADDED_IN_315
+        ),
+        required=True,
+    )
+
     class Meta:
-        root_type = "Fulfillment"
-        enable_dry_run = True
+        doc_category = DOC_CATEGORY_ORDERS
+        root_type = None
+        enable_dry_run = False
         interfaces = (Event,)
         description = "Event sent when new fulfillment is created." + ADDED_IN_34
 
+    @staticmethod
+    def resolve_fulfillment(root, info: ResolveInfo):
+        _, data = root
+        return data["fulfillment"]
+
+    @staticmethod
+    def resolve_order(root, info: ResolveInfo):
+        _, data = root
+        return data["order"]
+
```

Comment:
This can be resolved directly from fulfillment. No need to pass it as separate object.

Reply:
@zedzior I'm not sure if you are talking about resolving order in `resolve_fulfillment` or there is another place to resolve order. You are right, if fulfillment is resolved it is easy to resolve order from it directly, but for now I figured it out only by adding `resolve_order`. Should I look for  other place?